connect/src/com/telenor/connect/ui/ConnectLoginButton.java

```diff
@@ -125,6 +126,7 @@ private Uri getAuthorizeUri() {
         }
 
         parameters.put("state", connectStore.generateSessionStateParam());
+        parameters.put("lsi", ConnectSdk.getLogSessionId());
```

Comment:
ConnectLoginButton calls `authenticate`, so I guess we can remove this now. I guess we could/should move the same `"state"` logic (above call) to the authenticate method as well, but your choice if you want to do that now or not.

Reply:
nope, we cab't becauseChrome Custom Tabs does not call authenticate....
not super happy with the existing design, but you can put it in the tech debt list